VirtoCommerce.Storefront/Domain/Stores/StoreWorkContextBuilderExtensions.cs

```diff
@@ -21,13 +21,7 @@ public static Task WithStoresAsync(this IWorkContextBuilder builder, IEnumerable
             }
 
             builder.WorkContext.AllStores = stores.ToArray();
-            var currentStore = builder.HttpContext.GetCurrentStore(stores, defaultStoreId);
```

Comment:
Could you explain why did you remove these lines of code?

Reply:
Please read again, I already explained it:

> Your "fix" for store url breaks StoreUrlBuilder ToAppAbsolute and ToAppRelative methods, because ToAppRelative will never generate urls like ~/.... but instead always produce /...

Reply:
Reverted back my changes, will let you know if something broken